lib/build.js

```diff
@@ -14,6 +14,8 @@ var fs = require('graceful-fs')
   , mkdirp = require('mkdirp')
   , exec = require('child_process').exec
   , win = process.platform == 'win32'
+  , semver = require('semver')
+  , runtime = semver.parse(process.version).major < 1 ? 'node' : 'iojs'
```

Comment:
Can there be an override for this as an env variable? e.g. https://github.com/tcr/pangyp/commit/b9209b874ef8ac74111a389e16a8ea90978c70be

If I want to enable `gyp configure --target=1.3.0` *and* `gyp configure --target=0.12.0`, I'd rather not have to install both `iojs` and `node` to create this effect.